services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py

```diff
@@ -101,10 +101,6 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         description="If set to True the application will boot into debug mode",
     )
 
-    DYNAMIC_SIDECAR_REMOTE_DEBUG_PORT: PortInt = Field(
```

Comment:
you can also remove `DYNAMIC_SIDECAR_HOST` form the settings

Reply:
Done